web/models/user_from_auth.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Tudo.UserFromAuth do
+
+  def basic_info(auth) do
+    id = Map.get auth, :uid
+    info = Map.get auth, :info, %{}
+    avatar = Map.get info, :image
+
+    %{id: id, name: name_from_info(info), avatar: avatar}
+  end
+
+  defp name_from_info(info) do
+    case info do
+      %{name: name} when not is_nil(name) and name != "" ->
+        name
+    _ ->
+      name = [info.first_name, info.last_name]
```

Comment:
Most people who take GH seriously have a first and last name on their profile.
In the GH API however this is just `"name"` so we may need to _split_ on `' '` or simply store it as `"name"` ...